src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -263,6 +267,7 @@ def run_privacy_request(
         except BaseException as exc:  # pylint: disable=broad-except
             privacy_request.error_processing(db=session)
             # If dev mode, log traceback
+            log_graph_failure(failed_graph_analytics_event(privacy_request, exc))
             _log_exception(exc, config.dev_mode)
```

Comment:
Is there a reason we need two different writes to the log?

Reply:
`_log_exception` is just logging to the console. The new `log_graph_failure` that is added here tries to send high-level details when the graph fails to Fideslog so we can understand the types of failures that are happening across instances.

Reply:
I can rename the `log_graph_failure` to make it clear it's for fideslog.